kcqrs-example/build.gradle

```diff
@@ -31,12 +31,12 @@ repositories {
 dependencies {
   compile project(':kcqrs-appengine')
   compile project(':kcqrs-client-gson')
-  compile "org.jetbrains.kotlin:kotlin-stdlib-jre8:$kotlin_version"
+  compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
 
   providedCompile "com.google.appengine:appengine:$gae_version"
   compile "com.google.appengine:appengine-api-1.0-sdk:$gae_version"
   compile "com.google.appengine:appengine-tools-sdk:$gae_version"
-
+  compile 'com.sparkjava:spark-kotlin:1.0.0-alpha'
```

Comment:
I suggest to use compile("com.sparkjava:spark-core:2.7.1") spark core.
Spark-kotlin doesn't seem to be developed  since long time ago.

Reply:
https://github.com/perwendel/spark-kotlin/commit/d6ac9cbb3a6d9841581424e41c50be927e4ced72

Reply:
ok, just suggest